### DISCUSSION TITLE
Why do you remove contributions from the profile when leaving an organization or deleting a repository?

### DISCUSSION THREAD

**Discussion by yannxaver** [community/community#38568] [Other Feature Feedback, Questions, & Ideas]
This doesn't seem user-friendly. Is this a limitation of your current data structure or why do you decide to handle it like this?

**Comment by david-engelmann** [comment]
It's a fair point, Could you continue to associate the email address with your account and just make it not the primary email address?

**Reply by frankolious** [reply]
Contributions and Badges are work done, despite an organization going under or closing down. THese badges get us hired. Please reconsider removing them without our consent.

**Comment by charanquartz** [comment]
I am kinda wondering the same question.

**Comment by ossanna16** [comment]
Hi there @yannxaver and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
